src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,53 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
+	
+	/** A decorative prefix added to the beginning of lines printed by AddressBook */
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.